content/docs/how-we-work/release-versioning.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Release Versioning
+weight: 14
+description: We give attention to release versioning as it is the practice of assigning unique identifiers or version numbers to software releases, indicating changes, updates, or improvements made to the software. These version numbers typically follow a specific format or convention, allowing developers, users, and other stakeholders to understand the significance of each release and track the evolution of the software over time.
+---
+
+# Release Versioning
+
+We give attention to release versioning as it is the practice of assigning unique identifiers or version numbers to software releases, indicating changes, updates, or improvements made to the software. These version numbers typically follow a specific format or convention, allowing developers, users, and other stakeholders to understand the significance of each release and track the evolution of the software over time.
+
+We adhere to the [Semantic Versioning 2.0.0](https://semver.org/) format, in line with Drupal's practice, for release versioning. This method assigns version numbers to software releases, with each number representing major, minor, and patch updates. Following this standardized approach ensures clarity and consistency in communicating changes to our products, facilitating seamless collaboration and understanding among our team members and users.
```

Comment:
Do we use semantic versioning for our projects? I don't recall seeing this at all.

Reply:
Yes and that is the expected set with team as well.
Kindly check [MSF](https://axelerant.atlassian.net/projects/BMSF?selectedItem=com.atlassian.jira.jira-projects-plugin%3Arelease-page&status=released-unreleased) and [AMTS](https://axelerant.atlassian.net/projects/GLAM?selectedItem=com.atlassian.jira.jira-projects-plugin%3Arelease-page&status=released) for reference

Reply:
Sounds good.